terraform/applications/moderator/domain.tf

```diff
@@ -0,0 +1,17 @@
+# moderator.stage.mozit.cloud
+# -> lb, k8s
+
+# moderator.allizom.org
+# -> email / SES setup
+# -> lb, k8s
+
+module "moderator_mozit" {
+  source       = "github.com/mozilla-it/terraform-modules//aws/dns/subdomain?ref=master"
+  domain       = var.moderator_mozit
+  apex_zone_id = data.terraform_remote_state.k8s.outputs.stage_dns_zone_id
+}
+
+module "moderator_mozilla" {
```

Comment:
since we want to use the "front"-facing domain for SES (e.g. moderator.allizom.org), need to have it here for email setup & can't use CNAME in infoblox to just point it to moderator.stage.mozit.cloud. Also means we need it to be update-able by external-secrets (see PR description). This setup does break the circular dependency between terraform (managing zones, linking ELB to dns records) and kubernetes (creating ELBs whenever ingress-nginx updates)